### PR TITLE
chore: add unit test coverage to inspect report-tsi command

### PR DESCRIPTION
**Pull request by mcfarlm3**
Adds sample TSI data for use in testing `inspect` commands, and achieves 84% test coverage for the `inspect report-tsi` command. Some changes were also made to `report_tsi.go` to allow for testing of output, and for naming continuity across `inspect` commands.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: cmd/influxd/inspect/report_tsi/report_tsi.go [hunk 1]
```diff
@@ -400,23 +405,25 @@ func (report *reportTSI) printSummaryByMeasurement() error {
 		measurements = measurements[:n]
 	}
 
-	tw := tabwriter.NewWriter(os.Stdout, 8, 8, 1, '\t', tabwriter.AlignRight)
+	tw := tabwriter.NewWriter(cmd.OutOrStdout(), 8, 8, 1, '\t', tabwriter.AlignRight)
 
-	fmt.Fprintf(tw, "Summary\nDatabase Path: %s\nCardinality (exact): %d\n\n", report.enginePath, totalCardinality)
-	fmt.Fprint(tw, "Measurement\tCardinality (exact)\n\n")
+	fmt.Fprintf(cmd.OutOrStdout(), "Summary\nDatabase Path: %s\nCardinality (exact): %d\n\n", filepath.Join(report.dataPath, report.bucketId), totalCardinality)
+	fmt.Fprint(cmd.OutOrStdout(), "Measurement\tCardinality (exact)\n\n")
```
Comment (danxmoran) [inline cmd/influxd/inspect/report_tsi/report_tsi.go:411]: Should this one still be using `tw`?

File: cmd/influxd/inspect/report_tsi/report_tsi.go [hunk 2]
```diff
@@ -444,12 +451,12 @@ func (report *reportTSI) printShardByMeasurement(id uint64) error {
 	}
 
 	tw := tabwriter.NewWriter(os.Stdout, 8, 8, 1, '\t', 0)
-	fmt.Fprintf(tw, "===============\nShard ID: %d\nPath: %s\nCardinality (exact): %d\n\n", id, report.shardPaths[id], totalCardinality)
-	fmt.Fprint(tw, "Measurement\tCardinality (exact)\n\n")
+	fmt.Fprintf(cmd.OutOrStdout(), "===============\nShard ID: %d\nPath: %s\nCardinality (exact): %d\n\n", id, report.shardPaths[id], totalCardinality)
+	fmt.Fprint(cmd.OutOrStdout(), "Measurement\tCardinality (exact)\n\n")
```
Comment (danxmoran) [inline cmd/influxd/inspect/report_tsi/report_tsi.go:455]: Should this still use `tw`?

File: cmd/influxd/inspect/report_tsi/report_tsi_test.go [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+	cmd := initCommand(t, params)
+
+	if params.expectErr {
+		require.EqualError(t, cmd.Execute(), params.expectedOut)
+		return
+	}
+
+	// Get output
+	out := getOutput(t, cmd)
```
Comment (DStrand1) [inline cmd/influxd/inspect/report_tsi/report_tsi_test.go:182]: Here we are running the command twice for each test, is this necessary or could it be reduced to just once?

Comment (mcfarlm3) [reply cmd/influxd/inspect/report_tsi/report_tsi_test.go:182]: There are two places where the command is executed - but I don't think they would both happen during the same test run. One is run if an error is expected, and the other is run if output is expected. So I think it's okay as-is?

Comment (danxmoran) [reply cmd/influxd/inspect/report_tsi/report_tsi_test.go:182]: You could refactor `getOutput` to return `(string, error)` and remove the `require.NoError` from it, but I think it's ok as-is :+1:

**Review by danxmoran — APPROVED** [review]

**Pull request merged**